recipe/meta.yaml

```
@@ -6,36 +6,44 @@ package:
   version: {{ version }}
 
 source:
-  url: http://hepmc.web.cern.ch/hepmc/releases/{{ name }}-{{ version }}.tar.gz
-  sha256: 532c5387270dd55ff1e676e929a77c5910f2fba92fa62885b1f5ed0aef6fdde8
+  url: https://hepmc.web.cern.ch/hepmc/releases/{{ name }}-{{ version }}.tar.gz
+  sha256: 587faa6556cc54ccd89ad35421461b4761d7809bc17a2e72f5034daea142232b
   folder: source
+  patches:
+    - 0001-fallback-to-readlink-if-macOS-missing-greadlink.patch  # [osx]
 
 build:
   number: 0
```

```suggestion
  number: 1
```

The build number needs to get bumped given the checklist

> Bumped the build number (if the version is unchanged)